reference.page.jsx

```diff
@@ -24,7 +24,9 @@ export default function* () {
       let title = "";
       try {
         const match = titleRegexp.exec(content);
-        title = match[1].slice(0, -"documentation".length) + "- Deno Docs";
+        title =
+          match[1].slice(0, -"documentation".length).replace("&#x2F;", "/") +
```

Comment:
The approach is correct. Let's use the `unescape()` function from here https://jsr.io/@std/html/doc/~/unescape so that we unescape all of them and don't have to revisit this in the future.

Reply:
updated!